### ISSUE TITLE
?possible bug?

### ISSUE THREAD

**Issue by sasog23** [hasura/graphql-engine#5765]
hasura version 1.3.2

it looks like hasura always takes "AND", regardless ...
am i doing anything wrong

query MyQuery {
  identity_users(where: {_or: {first_name: {_ilike: "%dav%"}, last_name: {_ilike: "%dav%"}}}) {
    id
    full_name
    username
  }
}

Analyze
SELECT
  coalesce(json_agg("root"), '[]') AS "root"
FROM
  (
    SELECT
      row_to_json(
        (
          SELECT
            "_1_e"
          FROM
            (
              SELECT
                "_0_root.base"."id" AS "id",
                "identity"."user_full_name"("_0_root.base") AS "full_name",
                "_0_root.base"."username" AS "username"
            ) AS "_1_e"
        )
      ) AS "root"
    FROM
      (
        SELECT
          *
        FROM
          "identity"."users"
        WHERE
          (
            (
              ("identity"."users"."first_name") ILIKE (('%dav%') :: text)
            )
            AND (
              ("identity"."users"."last_name") ILIKE (('%dav%') :: text)
            )
          )
      ) AS "_0_root.base"
  ) AS "_2_root"

**Comment by tirumaraiselvan** [comment]
Please see the `Notes` here: https://hasura.io/docs/1.0/graphql/core/queries/query-filters.html#using-multiple-filters-in-the-same-query-and-or

TLDR: You need to pass an array of single key objects. This is definitely confusing and the DX of this is being tracked here: https://github.com/hasura/graphql-engine/issues/4488

**Comment by sasog23** [comment]
ok, got it

but then the problem is in the console, query was generated via. hasura console, by clicking fields and operators

query hasuraUser {
  users(where: {_or: [{first_name: {_ilike: "%dav%"}}, {last_name: {_ilike: "%dav%"}}]}) {
    id
    last_name
    first_name
    username
  }
}

**Comment by tirumaraiselvan** [comment]
Right, it's an issue with explorer (not developed by Hasura): https://github.com/hasura/graphql-engine/issues/4488#issuecomment-623073752
